sale_coupon_multi_gift/models/coupon_reward.py

```diff
@@ -17,18 +17,13 @@ def name_get(self):
         """Add complete description for the multi gift reward type."""
         res = super().name_get()
         for reward in self.filtered(lambda x: x.reward_type == "multi_gift"):
-            reward_string = _(
-                "Free Products - %s"
-                % (
-                    ", ".join(
-                        "{}x {}".format(
-                            reward.reward_product_quantity,
-                            fields.first(reward.reward_product_ids).name,
-                        )
-                        for reward in reward.coupon_multi_gift_ids
-                    )
+            reward_string = _("Free Products - %(name)s") % {
```

Comment:
Please modify at least the Spanish translation to get it automatically translated on merge.

Reply:
Done